.github/workflows/build-push-template.yml

```
@@ -7,30 +7,13 @@ on:
 
 jobs:
   build-push-template:
+    if: ${{ github.repository == 'substrate-developer-hub/substrate-node-template' }}
```

```suggestion
    if: ${{ github.repository == 'substrate-developer-hub/recipes' }}
```
If we keep this test, it should say recipes repo. But I wonder if we even need it. If I recall, we added this test to the node template, so that forks would not have a failing CI job right away. However the recipes is not intended as a template.

Thinking about it more, I guess people will fork the recipes to experiment on it, so I think we should leave this.